src/ManagedServices/ManagedServices/help/Get-AzManagedServicesAssignment.md

```diff
@@ -0,0 +1,232 @@
+
+
+Properties : Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Models.PSRegistrationAssignmentPropertiesRegistrationDefinitionProperties
+Plan       :
+Id         : /subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/providers/Microsoft.ManagedServices/registrationDefinitions/cae481c0-de7c-42a8-86c1-5b170861caf8
+Type       : Microsoft.ManagedServices/registrationDefinitions
+Name       : cae481c0-de7c-42a8-86c1-5b170861caf8
+```
+
+Gets a registration assignment with registration definition details.
+
+### Example 5
+```powershell
+PS C:\> Get-AzManagedServicesAssignment -Scope /subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/resourceGroups/newRG
+
+Name                                 RegistrationDefinitionId
+----                                 ------------------------
+c5deb1ba-8e27-4935-8af5-9242e7dabd24 /subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/providers/Microsoft.ManagedServices/registrationDefinitions/447b1aff-b0fc-4959-989d-d77dc93f3509
+aa891268-329a-4637-b3f6-2877ea304f8b /subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/providers/Microsoft.ManagedServices/registrationDefinitions/46b981a7-63ff-4063-9961-9fce4ddea376
+```
+
+Gets all the registration assignments.
+
+### Example 6
+```powershell
+PS C:\> $assignments = Get-AzManagedServicesAssignment
+PS C:\> $assignments[0].Id
+/subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/providers/Microsoft.ManagedServices/registrationAssignments/f2e18995-6c79-4ab7-876e-1b1c8393d12c
+PS C:\> Get-AzManagedServicesAssignment -ResourceId $assignments[0].Id
+
+Name                                 RegistrationDefinitionId
+----                                 ------------------------
+f2e18995-6c79-4ab7-876e-1b1c8393d12c /subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15/providers/Microsoft.ManagedServices/registrationDefinitions/a156aad7-f3ce-4a46-b240-246242b6bd78
+```
+
+Gets the registration assignment given the fully qualified resource id.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ExpandRegistrationDefinition
+Whether to include registration definition details.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Id
+The Registration Assignment identifier.
+
+```yaml
+Type: System.String
+Parameter Sets: ById
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+The Registration Assignment Id.
```

Comment:
I think here we need to be a bit more descriptive in disambiguating these two ids - it should be clear which one is a simple Guid, and which one is a resource Id (an example value in this description would likely be in order).

Reply:
updated to The Registration Assignment ResourceId. And added examples.